pyproject.toml

```
@@ -66,4 +67,10 @@ doc = [
 ]
 
 [tool.setuptools.packages.find]
-include = ['osier']
\ No newline at end of file
+include = ['osier']
+
+[tool.pytest.ini_options]
+addopts = "--cov"
+
+[tool.coverage.report]
+fail_under = 80
```

```suggestion
fail_under = 75
```

80% is a good goal but `osier` is currently at 76%! The coverage test is currently checking the coverage for the tests themselves... which is quite high, of course.